src/commands/wallet/create.ts

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
Instead of creating a new command (wallet), we should use the existent profile create.